src/snowflake/cli/api/commands/snow_typer.py

```diff
@@ -166,9 +163,14 @@ def post_execute(execution: ExecutionMetadata):
         Callback executed after running any command callable. Pay attention to make this method safe to
         use if performed operations are not necessary for executing the command in proper way.
         """
-        from snowflake.cli._app.telemetry import flush_telemetry, log_command_result
+        from snowflake.cli._app.telemetry import (
+            flush_telemetry,
+            log_command_result,
+            log_command_usage,
+        )
 
         log.debug("Executing command post execution callback")
+        log_command_usage(execution)
```

Comment:
does this mean that commands that do not require connection will not be logged anymore?